client/src/views/Comittees/FunctionHistory/index.tsx

```diff
@@ -1,22 +1,62 @@
-import { useContext } from 'react'
+import { useContext, useEffect, useRef, useState } from 'react'
+import ExportableTable from '../../../components/ExportableTable'
 import HeaderSecondary from '../../../components/Header/HeaderSecondary'
+import Table from '../../../components/Table'
+import Title from '../../../components/Title'
 import { EntityContext } from '../../../context/CommitteeContext'
+import { createPDF } from '../../../utils/CreatePDF'
+import { committee_details_mock } from '../../../_mock/comittee'
 
 const FunctionHistory = () => {
   const { currentEntity } = useContext(EntityContext)
+  const [exportPDF, setExportPDF] = useState(false)
+  const table = useRef<HTMLInputElement>(null)
+  const [comitteeContent, setComitteeContent] = useState<any[]>([
```

Comment:
It might be a good idea to standardize the spelling of committee into either the "comittee" misspelling or "committee"